public/batch.html

```diff
@@ -93,9 +93,9 @@ <h3 class="panel-title panel-title-batch">Direct input</h3>
 								<div class="form-group pull-right">
 									<div class="spinner" style="display: none"></div>
 									<button type="reset" name="reset-direct-input"
-										class="btn btn-link">Reset</button>
+										class="btn btn-default">Reset</button>
```

Comment:
The btn-default class changes the background color to white, providing sufficient contrast